test/test_core.py

```diff
@@ -106,6 +111,71 @@ def test_show_full_ontology_structure_without_schema():
     assert "lethal variant" in treevis
 
 
+def test_in_memory_kg_only_in_online_mode(core):
+    for in_memory_dbms in IN_MEMORY_DBMS:
+        core._dbms = in_memory_dbms
+        core._offline = True
```

Comment:
Shouldn't _offline here be False?

Reply:
Indeed, for correct behavior (online mode and in memory) _offline needs to be False. Nevertheless in this test it is tested if an error is raised and the user receives a meaningful error message, if the config is not compliant (e.g. an in memory DBMS is configured in offline mode and the user tries to get an in memory instance of the KG (which is not supported; this is only supported with the config in memory DBMS in online mode).

Reply:
Ah, I see, thank you

Reply:
No worries (: